modules/ztp-pre-staging-partitioning.adoc

```diff
@@ -0,0 +1,207 @@
+== Creating the partition
+
+Once the device is ready, you create a single partition and a GPT partition table.
+The partition is automatically labelled as `data` and created at the end of the device.
+Otherwise, the partition will be overridden by the `coreos-installer`.
+
+[IMPORTANT]
+====
+The `coreos-installer` requires the partition to be created at the end of the device and to be labelled as `data`. Both requirements are necessary to save the partition when writing the {op-system} image to the disk.
+====
+
+.Prerequisites
+
+* The container must run as `privileged` due to formatting host devices.
+* You have to mount the `/dev` folder so that the process can be executed inside the container.
+
+.Procedure
+
+In the following example, the size of the partition is 250 GB due to precaching Day 2 Operators for the DU profile.
+
+. Run the container as `privileged`:
++
+[source,terminal]
+----
+# podman run -v /dev:/dev --privileged --rm  \
+quay.io/openshift-kni/telco-ran-tools:latest -- \
+factory-precaching-cli partition -d /dev/nvme0n1 -s 250
+----
+
+. Check the storage information:
++
+[source,terminal]
+----
+# lsblk
+----
+
++
+.Example output
+[source,terminal]
+----
+NAME        MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
+loop0         7:0    0  93.8G  0 loop /run/ephemeral
+loop1         7:1    0 897.3M  1 loop /sysroot
+sr0          11:0    1   999M  0 rom  /run/media/iso
+nvme0n1     259:1    0   1.5T  0 disk
+└─nvme0n1p1 259:3    0   250G  0 part
+----
+
+.Verification
+
+You must verify that the following requirements are met:
+
+* The device has a GPT partition table
+* The partition uses the latest sectors of the device.
+* The partition is correctly labeled as `data`.
+
+. Query the disk status to verify that the disk is partitioned as expected:
++
+[source,terminal]
+----
+# gdisk -l /dev/nvme0n1
+----
+
++
+.Example output
+[source,terminal]
+----
+GPT fdisk (gdisk) version 1.0.3
+
+Partition table scan:
+  MBR: protective
+  BSD: not present
+  APM: not present
+  GPT: present
+
+Found valid GPT with protective MBR; using GPT.
+Disk /dev/nvme0n1: 3125627568 sectors, 1.5 TiB
+Model: Dell Express Flash PM1725b 1.6TB SFF
+Sector size (logical/physical): 512/512 bytes
+Disk identifier (GUID): CB5A9D44-9B3C-4174-A5C1-C64957910B61
+Partition table holds up to 128 entries
+Main partition table begins at sector 2 and ends at sector 33
+First usable sector is 34, last usable sector is 3125627534
+Partitions will be aligned on 2048-sector boundaries
+Total free space is 2601338846 sectors (1.2 TiB)
+
+Number  Start (sector)    End (sector)  Size       Code  Name
+   1      2601338880      3125627534   250.0 GiB   8300  data
+----
+
+[id="ztp-mount-partition_{context}"]
+== Mounting the partition
+
+After verifying that the disk is partitioned correctly, you can mount the device into `/mnt`.
+
+[IMPORTANT]
+====
+It is recommended to mount the device into `/mnt` because the mounting point is used during ZTP preparation.
```

Comment:
It is recommended to mount the device into `/mnt` because **that or the?** mounting point is used during ZTP preparation.